docker-images/kafka/scripts/kafka_run.sh

```diff
@@ -47,6 +43,26 @@ echo "Starting Kafka with configuration:"
 ./kafka_config_generator.sh | tee /tmp/strimzi.properties | sed -e 's/sasl.jaas.config=.*/sasl.jaas.config=[hidden]/g' -e 's/password=.*/password=[hidden]/g'
 echo ""
 
+# Prepare for Kraft
+if [ -n "$CLUSTER_ID" ]; then
```

Comment:
Maybe we should have an explicit var for enabling ZK in the scripts. Or rename it so it's `KRAFT_CLUSTER_ID`